src/features/auth/components/sign-in-side/index.tsx

```diff
@@ -0,0 +1,164 @@
+  password: string;
+};
+
+const theme = createTheme();
+
+export default function SignInSide() {
+  const [formResult, setFormResult] = useState({email: '', password: ''});
+  const [loginForm, {}] = useMutation(LOGIN);
+  const router = useRouter();
+
+  const emptyForm: AuthForm = {
+    email: '',
+    password: '',
+  };
+
+  const form = useForm<AuthForm>({
+    defaultValues: emptyForm,
+  });
+
+  const sendGetInTouch = useAsync(async () => {
+    const formValues = form.getValues();
+    await loginForm({
+      variables: {
+        input: {
+          email: formValues.email,
+          password: formValues.password,
+        },
+      },
+    });
+  }, false);
+
+  const {watch} = form;
+
+  const handleCredentialsSubmit = async () => {
+    const {email, password} = formResult;
+    const res = await logIn(email, password);
+    if (res.status === 200) {
+      router.push('/home');
+    } else {
+      toast('Internal Server Error: ' + res.response.status);
+    }
+  };
+
+  // Effect to handle sendGetInTouch request status
+  React.useEffect(() => {
+    if (sendGetInTouch.status === 'success') {
+      form.reset(emptyForm);
+      toast.success('Thank you for your message!');
+    }
+    if (sendGetInTouch.status === 'error') {
+      toast.error(sendGetInTouch.error);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [sendGetInTouch.status]);
+
+  function handleChange(event: any) {
+    const {value, name} = event.target;
+
+    form.setValue(name, value);
+    form.clearErrors([name]);
+    setFormResult({...formResult, [name]: value});
+  }
+
+  return (
+    <ThemeProvider theme={theme}>
+      <Grid container component='main' sx={{height: '100vh', minHeight: '100vh'}}>
+        <CssBaseline />
+        <Grid
+          item
+          xs={false}
+          sm={4}
+          md={7}
+          sx={{
+            backgroundImage: 'url(/collaboration.jpeg)',
+            backgroundRepeat: 'no-repeat',
+            backgroundColor: '#5d5a88',
+            backgroundSize: 'cover',
+            backgroundPosition: 'center',
+          }}
+        />
+        <Grid item xs={12} sm={8} md={5} component={Paper} elevation={6} square>
+          <Box
+            sx={{
+              my: 8,
+              mx: 4,
+              display: 'flex',
+              flexDirection: 'column',
+              alignItems: 'center',
+            }}
+            className={styles.form}
+          >
+            <Avatar sx={{m: 1, bgcolor: 'secondary.main'}}>
+              <ArrowBack fontSize={'medium'} />
+            </Avatar>
+            <Typography component='h1' variant='h5' className={styles.title}>
+              Login
+            </Typography>
+            <form className={styles.form}>
+              <TextField
+                margin='normal'
+                fullWidth
+                id='email'
+                label='Email Address'
+                variant='standard'
+                name='email'
```

Comment:
Personally, I would prefer to move this into a CSS file to keep consistency over the project :D

Reply:
que parte? hay cosas que estan el css y otras que no pero por gusto. Es parte de la gracia de material ui para hacer algunas cosas desde aca mas facil con una prop en un component

Reply:
coco

Reply:
No CSS in js, or at least the least amount of it. Cmon, do not make me call to fabi

Reply:
Change it!